components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationMgtDBQueries.java

```diff
@@ -21,8 +21,8 @@
 /**
  * This class contains default SQL queries
  * <p/>
- * TODO : Make the queries configurable from a file TODO : Use transactions and joins
  */
+@Deprecated
```

Comment:
In the doc comment can we say SQL queries are internal implementation of the DAO layer and should not be exposed to components outside.